src/univariate/continuous/noncentralf.jl

```diff
@@ -56,8 +56,6 @@ function rand(rng::AbstractRNG, d::NoncentralF)
     r1 / r2
 end
 
-# TODO: remove RFunctions dependency once NoncentralChisq has its removed
-@rand_rdist(NoncentralF)
 function rand(d::NoncentralF)
```

Comment:
Similar to above, this has to be changed to
```suggestion
function rand(rng::AbstractRNG, d::NoncentralF)
```